mocks/README.md

```diff
@@ -0,0 +1,56 @@
+# Adding a router
+
+1. Add your endpoint in `/routes`. See example 1 below:
+```javascript
+  // Example 1
+  {
+    id: 'get-wpi-aanvragen',  // <--- let's call this value a
+    url: `${settings.MOCK_BASE_PATH}/wpi-koppel-api/wpi/uitkering/aanvragen`,
+    method: 'GET',
+    variants: [
+      {
+        id: 'standard',  // <-------------------- and this b
+        type: 'json',
+        options: {
+          status: 200,
+          body: RESPONSES.AANVRAGEN,
+        },
+      },
+    ],
+  },
+```
+2. Add the id inside `collections.json` of form `a:b` (see example 1 above)
+3. Add to the `hit-endpoints.js` test script.
+
+
+# Adding to the test script
+
+When adding a route with custom logic like with [Middlware][1] or with a your own [Variant Handler][2]...<br>
+Or a complex route containing something like `/:profile(user|company)`.
+Please also create a way to hit this endpoint in `CUSTOM_REQUESTS` which is found in the `hit-all-endpoints.js` script.<br>
+This is because it cannot automatically figure out how to hit this, like with a simple 'get json in all cases' endpoint.
```

Comment:
consider change the text into more readable: 
- This is necessary because the script cannot automatically determine how to interact with such endpoints, unlike simpler 'GET JSON in all cases' endpoints.

Reply:
dat leest veel makkelijker inderdaad, bedankt